.github/ISSUE_TEMPLATE/bug-task.yaml

```diff
@@ -0,0 +1,70 @@
+name: Hotham Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
+      options:
+        - 0.1.1 (Default)
+        - latest
+    validations:
+      required: true
+  - type: dropdown
+    id: hotham-component
+    attributes:
+      label: In which Hotham component are you seeing the problem on?
+      multiple: true
+      options:
+        - Core
+        - Example
+        - Simulator
+        - Debug 
+        - Other
+  - type: dropdown
+    id: vr-system
+    attributes:
+      label: What VR System are you seeing the problem on?
+      multiple: false
+      options:
+        - Quest-2
+        - Quest-1
```

Comment:
@davidkuhta Whoops! I think I meant a dropdown for VR system!

Reply:
The VR selection is a dropdown?